README.md

```diff
@@ -127,64 +127,56 @@ After running the above command, the pet store web service will be auto-generate
 #### Generated `ballerina_petstore.bal` file
   
 ```ballerina
-import ballerina/log;
 import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
 import ballerina/swagger;
 
-endpoint http:Listener ep0 { 
-    host: "localhost",
-    port: 9090
-};
+listener http:Listener ep0 = new(9090);
 
-@swagger:ServiceInfo { 
+@swagger:ServiceInfo {
     title: "Ballerina Petstore",
-    description: "This is a sample Petstore server.",
+    description: "This is a sample Petstore server. This uses swagger definitions to create the ballerina service",
     serviceVersion: "1.0.0",
     termsOfService: "http://ballerina.io/terms/",
     contact: {name: "", email: "samples@ballerina.io", url: ""},
     license: {name: "Apache 2.0", url: "http://www.apache.org/licenses/LICENSE-2.0.html"},
     tags: [
-        {name: "pet", description: "Everything about your Pets", externalDocs:
-        { description: "Find out more", url: "http://ballerina.io" } }
+        {name: "pet", description: "Everything about your Pets", externalDocs: {description: "Find out more", url: "http://ballerina.io"}}
     ],
-    externalDocs: { description: "Find out about Ballerina", url: "http://ballerina.io" },
-    security: [
-    ]
+    externalDocs: {description: "Find out more about Ballerina", url: "http://ballerina.io"}
 }
 @http:ServiceConfig {
     basePath: "/v1"
 }
-service BallerinaPetstore bind ep0 {
+service BallerinaPetstore on ep0 {
 
     @swagger:ResourceInfo {
         summary: "Update an existing pet",
         tags: ["pet"]
     }
-    @http:ResourceConfig { 
+    @http:ResourceConfig {
         methods:["PUT"],
         path:"/pet",
-        body:"petDetails"
+        body:"_updatePetBody"
     }
-    updatePet (endpoint outboundEp, http:Request req, Pet petDetails) {
-        http:Response res = updatePet(req, petDetails);
-        outboundEp->respond(res) but { error e => log:printError("Error while responding",
-            err = e) };
+    resource function updatePet (http:Caller outboundEp, http:Request _updatePetReq, Pet _updatePetBody) {
+        http:Response _updatePetRes = updatePet(_updatePetReq, _updatePetBody);
+        _ = outboundEp->respond(_updatePetRes);

```

Comment:
Shall we handle the error in all the places without ignoring it?

        var result = outboundEp->respond(res);

         if (result is error) {
            log:printError("Error while responding", err = result);
        }

Reply:
Ahh yes. I handled at the source files, but not in the README. Will do